File: otter/step0_repl.py
def READ(line: str) -> str:
    return line

def EVAL(line: str) -> str:
    return line

def PRINT(line: str) -> str:
    return line

def rep(line: str) -> str:
    re = READ(line)
    ev = EVAL(re)
    return PRINT(ev)

while(True):
    line = input('user> ')
    print(rep(line))

File: pale/step0_repl.js
const readlineSync = require('readline-sync');

function main() {
    while (true) {
        var input = readlineSync.question("user> ");
            rep(input);
    }
}

function READ(input) { return input }
function EVAL(input) { return input }
function PRINT(input) { return input }
function rep(input) {
    console.log(PRINT(EVAL(READ(input))));
}

main();